packages/react-bootstrap-table2/test/row.test.js

```diff
@@ -442,128 +442,180 @@ describe('Row', () => {
-    });
-  });
 
-  describe('when selectRow.mode is checkbox or radio (row was selectable)', () => {
+  describe('selectRow', () => {
     let selectRow;
-    beforeEach(() => {
-      selectRow = { mode: 'checkbox' };
-      wrapper = shallow(
-        <Row
-          { ...mockBodyResolvedProps }
-          rowIndex={ rowIndex }
-          columns={ defaultColumns }
-          row={ row }
-          selectRow={ selectRow }
-          selected
-          selectable
-        />);
-    });
 
-    it('should rendering SelectionCell component correctly', () => {
-      expect(wrapper.find(SelectionCell).length).toBe(1);
-    });
-
-    it('should render SelectionCell component with correct props', () => {
-      expect(wrapper.find(SelectionCell).props().selected).toBeTruthy();
-      expect(wrapper.find(SelectionCell).props().disabled).toBeFalsy();
-      expect(wrapper.find(SelectionCell).props().mode).toEqual(selectRow.mode);
-    });
-
-    describe('if selectable prop is false', () => {
+    describe('when selectRow.mode is ROW_SELECT_DISABLED (row is not able to select)', () => {
       beforeEach(() => {
         wrapper = shallow(
           <Row
             { ...mockBodyResolvedProps }
             rowIndex={ rowIndex }
             columns={ defaultColumns }
             row={ row }
-            keyField={ keyField }
-            selectRow={ selectRow }
-            selectable={ false }
-          />);
+          />
+        );
       });
 
-      it('should render SelectionCell component with correct disable prop correctly', () => {
-        expect(wrapper.find(SelectionCell).length).toBe(1);
-        expect(wrapper.find(SelectionCell).prop('disabled')).toBeTruthy();
+      it('should not render <SelectionCell />', () => {
+        expect(wrapper.find(SelectionCell).length).toBe(0);
       });
     });
 
-    describe('if selectable prop is true', () => {
-      beforeEach(() => {
-        wrapper = shallow(
-          <Row
-            { ...mockBodyResolvedProps }
-            rowIndex={ rowIndex }
-            columns={ defaultColumns }
-            row={ row }
-            keyField={ keyField }
-            selectRow={ selectRow }
-            selectable
-          />);
-      });
+    describe('when selectRow.mode was defined  (single or multiple selection)', () => {
+      describe('if selectRow.mode is radio (single selection)', () => {
+        beforeEach(() => {
+          selectRow = { mode: 'radio' };
+          wrapper = shallow(
+            <Row
+              { ...mockBodyResolvedProps }
+              rowIndex={ rowIndex }
+              columns={ defaultColumns }
+              row={ row }
+              selectRow={ selectRow }
+              selected
+              selectable
+            />);
+        });
+
+        it('should render <SelectionCell />', () => {
+          expect(wrapper.find(SelectionCell).length).toBe(1);
+        });
+
+        it('should render <SelectionCell /> with correct props', () => {
```

Comment:
Nice, you are suppose to check those prop, that's something you missing before..